security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -10,11 +10,71 @@ The {cert-manager-operator} is a cluster-wide service that provides application
 
 These release notes track the development of {cert-manager-operator}.
 
-:FeatureName: The {cert-manager-operator}
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.10.2"]
+== Release notes for {cert-manager-operator} 1.10.2
+
+// TODO: Confirm date closer to GA
+Issued: 2023-03-23
+
+The following advisory is available for the {cert-manager-operator} 1.10.2:
+
+* link:https://access.redhat.com/errata/RHEA-2023:1238[RHEA-2023:1238]
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10[cert-manager project release notes for v1.10].
+
+[IMPORTANT]
+====
+If you used the Technology Preview version of the {cert-manager-operator}, you must uninstall it and move all related resources for the Technology Preview version before installing this version of the {cert-manager-operator}.
+
+For more information, see xref:../../security/cert_manager_operator/cert-manager-operator-uninstall.adoc#cert-manager-operator-uninstall[Uninstalling the {cert-manager-operator}].
+====
+
+[id="cert-manager-operator-1.10.2-new-features-and-enhancements"]
+=== New features and enhancements
+
+This is the general availability (GA) release of the {cert-manager-operator}.
+
+* The following issuer types are supported:
+** ACME
+** CA
+** Self-signed
+
+* The following ACME challenge types are supported:
+** DNS-01
+** HTTP-01
+
+* The following DNS-01 providers for ACME issuers are supported:
+** Amazon Route 53
+** Azure DNS
+** Google Cloud DNS
+
+* The {cert-manager-operator} now supports injecting custom CA certificates for the cluster-wide egress proxy configuration.
```

Comment:
@bergerhoffer @thejasn need it mention `supports injecting egress proxy env vars automatically` as well?

Reply:
This gets documented as across all operators https://docs.openshift.com/container-platform/4.12/operators/admin/olm-configuring-proxy-support.html, probably we can call out that this is supported now.

Reply:
I added "and egress proxy environment variables" - let me know if that sounds okay.

Reply:
["and propagates cluster wide egress proxy environment variables" ](https://github.com/openshift/openshift-docs/pull/56754/files#r1142397268)